.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,23 @@
+name: Lint Python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
```

Comment:
what is the default python version?

Reply:
Python 3.10.0 and allowing it to float means that our tests will find incompatibilities in new Python releases before our users do.